content/docs/user_guide/install.md

```diff
@@ -0,0 +1,162 @@
++++
+title = "Installation"
+weight = 20
++++
+
+> Note: gVisor supports only x86\_64 and requires Linux {{< required_linux >}}
+> ([older Linux][old-linux]).
+
+## Versions
```

Comment:
I'm not really sure that this section is something the folks who install via 'apt' need to think about. Structurally, consider combining this with the "Manually" section.